app/src/main/java/com/hammer/talkbbokki/presentation/navigation/TalkbbokkiNavHost.kt

```diff
@@ -61,7 +63,7 @@ fun TalkbbokkiNavHost(
             navigateToDetail = { level, item ->
                 navController.navigate(
                     DetailDestination.route +
-                            "?level=$level&id=${item.id}&tag=${item.tag}&topic=${item.name}&shareLink=${item.shareLink}"
+                            "?level=$level&id=${item.id}&tag=${item.tag}&topic=${item.name}&shareLink=${item.shareLink}&bgColor=${item.bgColor}"
```

Comment:
🚫 **[ktlint]** <indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (28) (should be 24)